### ISSUE TITLE
As a road traveller, I would like that map.clean-roads.eu is able to automatically update the data visualized on the map with the last measurements available, so that I am informed about possible issues on the road and I can drive more safely

### ISSUE THREAD

**Issue by rcavaliere** [tis-innovation-park/cleanroads-map#2]
We should foresee a way for the user to check if there new data records available. This could be done manually through a specific icon or automatically through an automatic update process.

**Comment by bertolla** [comment]
Does this make sense for a browser or a mobile?
If the timeinterval would be smaller, like every 10 seconds, it might make sense, but nobody waits for 10 minutes in front of a website.

Is there a usecase?

**Comment by rcavaliere** [comment]
Yes, the webapp could be used even by specialized users just to check very quickly if there is some event on the road network. It could happen that e.g. road operators leave the application open on a browser, start other activities and then come back to the monitor to check the situation.

**Comment by rcavaliere** [comment]
I don't see this functionality implemented.
